src/main/java/org/folio/edge/sip2/utils/TenantUtils.java

```diff
@@ -15,63 +15,76 @@ private TenantUtils() {
+   * @param sip2config - SIP2 edge module config.
+   *     Contains a JSON array (scTenants) of tenant configs,
    *     each with a scSubnet element with a CIDR-notation string value.
-   *    
+   *     <p>
    *     I.E:<br>
-   * <pre>
-       {
-        "scTenants": [
-          {
-            "scSubnet": "11.11.00.00/16",
-            "tenant": "diku",
-            "errorDetectionEnabled": true,
-            "messageDelimiter": "\r",
-            "charset": "ISO-8859-1"
-          },
-          {
-            "scSubnet": "22.22.00.00/16",
-            "tenant": "test_tenant",
-            "errorDetectionEnabled": true,
-            "messageDelimiter": "\r",
-            "charset": "ISO-8859-1"
-          },
-          {
-            "scSubnet": "33.33.00.00/16",
-            "tenant": "test_tenant",
-            "errorDetectionEnabled": true,
-            "messageDelimiter": "\r",
-            "charset": "ISO-8859-1"
-          }
-        ]
-      }
-   * </pre>
-   * 
+   *     <pre>
+   *     {
+   *      "scTenants": [
+   *        {
+   *          "scSubnet": "11.11.00.00/16",
+   *          "tenant": "diku",
+   *          "errorDetectionEnabled": true,
+   *          "messageDelimiter": "\r",
+   *          "charset": "ISO-8859-1"
+   *        },
+   *        {
+   *          "scSubnet": "22.22.00.00/16",
+   *          "tenant": "test_tenant",
+   *          "errorDetectionEnabled": true,
+   *          "messageDelimiter": "\r",
+   *          "charset": "ISO-8859-1"
+   *        },
+   *        {
+   *          "scSubnet": "33.33.00.00/16",
+   *          "tenant": "test_tenant",
+   *          "errorDetectionEnabled": true,
+   *          "messageDelimiter": "\r",
+   *          "charset": "ISO-8859-1"
+   *        }
+   *      ]
+   *     }
+   *     </pre>
+   *     </p>
    * @param clientIP - IPv4 address of client SC used as lookup to find tenant config
-   * @returns tenant config whose scSubnet encompasses clientIP. 
-   *     Returns the sip2conf itself if it does not contain a scTenants element or 
-   *     the scTenants array has no tenant with subnet in range for clientIP. 
+   * @param port Tenant specific port value.
+   * @returns tenant config whose scSubnet encompasses clientIP.
+   *     Returns the sip2conf itself if it does not contain a scTenants element or
+   *     the scTenants array has no tenant with subnet in range for clientIP.
    */
-  public static JsonObject lookupTenantConfigForIPaddress(JsonObject sip2config, String clientIP) {
-    
+  public static JsonObject lookupTenantConfigForIPaddress(JsonObject sip2config, String clientIP,
+                                                          int port) {
+
     if (!sip2config.containsKey(SC_TENANTS)) {
       log.debug("LookupTenantConfig scTenants key not found in config, "
           + "support for muti-tenant not available");
       return sip2config;
     }
-    
+
     Optional<JsonObject> tcOpt = sip2config.getJsonArray(SC_TENANTS).stream()
```

Comment:
I prefer longer variable names like `tenantConfigOptional` (know you're inheriting this from old code).

Reply:
Done